food-truck-frontend/src/api/RouteLocationApi.ts

```diff
@@ -1,53 +1,38 @@
 import api from "../util/api";
-import {backendToFrontend, frontendToBackend, RouteLocation,} from "../components/map/route-map/RouteLocation";
+import {backendToFrontend, frontendToBackend, RouteLocation, RouteLocationMeta,} from "../domain/RouteLocation";
 import DayOfWeek from "../components/map/route-map/DayOfWeek";
+import {parse} from "../util/type-checking";
 
-export const loadTodaysRoute = async (
+export const loadCurrentRoute = async (
   truckId: number,
   onFail?: (res: any) => void
-) => {
+): Promise<RouteLocation[] | null> => {
   let routePts: RouteLocation[] = [];
   let nextStopId: number = 1;
 
-  await api
-    .request({
-      url: `/truck/${truckId}/active-route`,
-      method: "GET",
-      params: {
-        now: new Date()
-      }
-    })
-    .then((response) => {
-      if (response.data)
-        routePts = response.data.locations.map((pt: any) =>
-          backendToFrontend(pt, nextStopId++)
-        );
-    })
-    .catch(onFail);
-  return routePts;
+  const resp = await api.get(`/truck/${truckId}/active-route`);
```

Comment:
So we're no longer passing the current date/time? Was the backend updated accordingly?